hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -145,6 +193,14 @@
 	  name: "bat_disable",
 	  desc: "0: don't invert; 1: invert",
 	}
+        { bits: "10",
+	  name: "lid_open",
+	  desc: "0: don’t invert; 1: invert",
+	}
+        { bits: "11",
+	  name: "z3_wakeup",
+	  desc: "0: don’t invert; 1: invert",
```

Comment:
and here

Reply:
Okay. copy-and-paste from bat_disable desc.